### ISSUE TITLE
Bug - P4G: Failed to obtain port for Reloaded Instance with community enhancement pack

### ISSUE THREAD

**Issue by SmartassSkeleton** [Reloaded-Project/Reloaded-II#178]
**Did you install the Prerequisites?**

I have installed all the prerequisites using the setup phase of the [CEP installation guide](https://p4g.cep.one/download-and-prepare#automatic-download-and-installation), as well ass downloaded the setup.exe file from this repo's releases page and ran it. As I understand this should be enough?

**Describe the Issue**
After opening Reloaded and setting my P4G path in the configuration I get this log:
```
[15:58:01] [Reloaded] Create Loader | Time: 150ms
[15:58:01] [Reloaded] Loaded: reloaded.sharedlib.hooks in 15ms
[15:58:01] [Reloaded] Current Process Location: C:\Program Files (x86)\Steam\steamapps\common\Persona 4 Golden\P4G.exe
[15:58:01] [modloader] Persona 4 Golden (Steam) Mod loader by TGE (2020) v1.4.3
[15:58:01] [R-II SteamHook] SteamAPI_RestartAppIfNecessary hooked successfully.
[15:58:01] [R-II SteamHook] SteamAPI_IsSteamRunning hooked successfully.
[15:58:01] [Reloaded] Loaded: p4gpc.modloader in 155ms
```
And this pops up:
![image](https://user-images.githubusercontent.com/8848843/210383602-89a718a9-09db-4d27-9b3e-be424a87ca1e.png)

I've tried to follow the suggestions in the warning but non of them helped. After ~10 times of trying to relaunch the game it managed to run and the intro skip mod (Tiny Fixes, the one provided by CEP) worked as expected. I then tried to run it a 2nd time but I encountered the same problem. I have no antivirus running so no exceptions needed.

After the game ran once I decided to proceed and run Aemulus and try and launch the game through there. So I set up the paths and unpack the base files, then I proceed to build my loadout. The game ran 1st try with the CEP mods on and built. 

I then proceeded to try and run the game again using Reloaded and I couldn't reproduce the error again.

Related issues: #170

**Comment by Sewer56** [comment]
Did the log terminate on that specific line? Sounds like some mod got stuck initialising for one reason or another.

Normally you can manually trigger this by highlighting the console window, and thus stalling mod loader startup but I doubt you could have done that, which makes me curious.

**Comment by SmartassSkeleton** [comment]
Thanks for the prompt reply!

When I get the warning message I usually just close the command line window manually and try again later after trying to research help online. I've attached all the I've had in the logs folder: [logs.zip](https://github.com/Reloaded-Project/Reloaded-II/files/10344663/logs.zip)

I don't think I've highlighted anything in the console, at least not ~10 times.

**Comment by RocksInMyDryer** [comment]
I'm having the same issue, I believe. I've followed each step of the CEP guide, up to https://p4g.cep.one/reloaded-ii

But every single time I click "Launch Application" in Reloaded II, I get:
![image](https://user-images.githubusercontent.com/44074878/213096923-01c28acc-a13d-4d73-93aa-1a091ca11457.png)

I added the Modding folder to my Avast Antivirus exceptions which doesn't seem to help. I also tried running Reloaded II as an administrator, but that made no difference either.
